src/upload/package.rs

```diff
@@ -23,16 +25,30 @@ impl Package {
             Ok(self.main.clone())
         }
     }
+    pub fn module(&self, package_dir: &PathBuf) -> Result<PathBuf, failure::Error> {
+        if self.module == PathBuf::from("") {
+            failure::bail!(
+                "The `module` key in your `package.json` file is required when using the module script format; please specify the entry point of your Worker.",
+            )
+        } else if !package_dir.join(&self.module).exists() {
+            failure::bail!(
+                "The entrypoint of your Worker ({}) could not be found.",
+                self.module.display()
```

Comment:
Is `display` required here?

Reply:
I believe so, I don't think `Path` implements `Display` on its own.